server/routes/transactions/index.ts

```diff
@@ -50,23 +47,22 @@ export default function routes(services: Services): Router {
       title: 'Transactions',
       config: getConfig(),
       data: {
-        accountTypes,
+        accountTypes: AccountTypes,
         balance: getBalanceByAccountCode(balances, accountCode),
-        contentHubTransactionsHelpLinkUrl: `${prisonerContentHubURL}/content/8534`,
+        contentHubTransactionsHelpLinkUrl: `/content/8534`,
```

Comment:
We may have to consider linking to the appropriate language on content hub here going forward? Do we link directly to Content Hub content a lot?

Reply:
Yeh quite a few places, but we get this url in middleware so shouldn't have to do much to update based on language 👍